### PR TITLE
Frontend improvement

### PR DESCRIPTION
**Pull request by meets2tarun**
#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia) and my PR follows them.
- [x] There is a corresponding issue for this pull request.
- [x] My branch is up-to-date with the Upstream `master` branch.
- [x] The unit tests pass locally with my changes (provide a screenshot or link for test) 
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)

Fix#121

### CODE REVIEW

File: templates/index.html [hunk 1]
```diff
@@ -12,7 +12,6 @@
   <link href='https://fonts.googleapis.com/css?family=Roboto:400,100' rel='stylesheet' type='text/css'>
   <script src="{{ url_for('static', filename='js/jquery.min.js') }}"></script>
   <script src="{{ url_for('static', filename='js/bootstrap.min.js') }}"></script>
-  <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/js/bootstrap.min.js"></script>
```
Comment (xeon-zolt) [inline templates/index.html]: still thinking why we had a multiple bootstrap js

**Comment by xeon-zolt** [comment]
@meets2tarun always provide with a link or screenshot for the UI changes showing what you have changed

**Comment by meets2tarun** [comment]
Sure @xeon-zolt , I'll take care of the same.
Please ship this.

screen shot 2018-05-27 at 18 58 16

Thanks

**Comment by meets2tarun** [comment]
@xeon-zolt Sure, I'll take care of this.

screen shot 2018-05-27 at 18 59 44


Thanks

**Comment by xeon-zolt** [comment]
@meets2tarun let's focus on functionality first then we can focus on improving the UI

**Comment by meets2tarun** [comment]
@xeon-zolt 
Sure Harsh,
We have a meeting down there this Wednesday 
Please do come and I propose an idea in the blog3 (discussed between me and Nicco).

**Comment by xeon-zolt** [comment]
cannot come on a weekday (only available late in the evening ) 
what about weekend ??

**Comment by meets2tarun** [comment]
@xeon-zolt 
Give your desired time here:
https://dudle.inf.tu-dresden.de/meilix-meet-16/

**Pull request merged**